domain/port/state/state_test.go

```diff
@@ -45,7 +44,7 @@ var _ = gc.Suite(&stateSuite{})
 var (
 	machineUUIDs = []string{"machine-0-uuid", "machine-1-uuid"}
 	netNodeUUIDs = []string{"net-node-0-uuid", "net-node-1-uuid"}
-	appNames     = []string{"app-0-name", "app-1-name"}
+	appNames     = []string{"appzero", "appone"}
 )
```

Comment:
**thought:** looks like my question about the application pattern which doesn't allows single digit surrounded by hyphen add consequences.

Reply:
Yep. Looks like someone (me) accidentally hard coded an invalid application name into a test.

As a side, idk why I went for `appzero` and not the much clearer `app-zero`